crates/teloxide/examples/deep_linking.rs

```diff
@@ -0,0 +1,133 @@
+use dptree::{case, deps};
```

Comment:
It will be good to have a top-level overview here which explains essential parts of the example (like in the `composite_state` one)

Reply:
Is the explanation ok?